### PR TITLE
DAOS-5696 test: Update jobtest utility

### PR DESCRIPTION
**Pull request by mjmac**
Add support for pool labels and more test modes.

Signed-off-by: Michael MacDonald <mjmac.macdonald@intel.com>

### CODE REVIEW

File: src/tests/jobtest.c [hunk 1]
```diff
@@ -245,55 +258,65 @@ main(int argc, char **argv)
...
+		for (i = 0; i < num_pools; i++) {
+			int j;
+
+			for (j = 0; j < handles_per_pool; j++) {
+				uuid_t c_uuid;
+				char cstr[64];
+
+				uuid_generate(c_uuid);
+				printf("Creating container using handle %" PRIu64 "\n",
+				pool_handles[i][j].cookie);
```
Comment (kjacque) [inline src/tests/jobtest.c:273]: Nit - incorrect indentation

File: src/tests/jobtest.c [hunk 2]
```diff
@@ -245,55 +258,65 @@ main(int argc, char **argv)
...
 
+	/* Use our handles */
+	if (use_handles) {
+		printf("\n");
+		for (i = 0; i < num_pools; i++) {
+			int j;
+
+			for (j = 0; j < handles_per_pool; j++) {
+				uuid_t c_uuid;
+				char cstr[64];
```
Comment (kjacque) [inline src/tests/jobtest.c:269]: I think we have a DAOS_UUID_STR_LEN constant declared somewhere

**Review by kjacque — COMMENTED** [review]
Changes look reasonable to me. A couple minor comments.

**Comment by kprantis** [comment]
Is the ticket number right for this? Seems to be a long resolved ticket? Making it hard to judge if it should be prioritized in CI queue or no...

**Comment by mjmac** [comment]
> 
> 
> Is the ticket number right for this? Seems to be a long resolved ticket? Making it hard to judge if it should be prioritized in CI queue or no...

It should not be prioritized. This is a patch for 2.2, consisting of some updates made for the MS163 demo. I just picked the ticket number that introduced the utility, which is only used for testing the auto-cleanup code. :)

**Comment by mjmac** [comment]
@kjacque, @dpquigl: This should be ready to land now. I wasn't paying attention to it for a while and it got a little bit-rotten.

**Pull request merged**